staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -257,6 +269,24 @@ func (w *watchCache) processEvent(event watch.Event, resourceVersion uint64, upd
 	return updateFunc(elem)
 }
 
+// Send a bookmark event with the current resourceVersion
+func (w *watchCache) sendBookmark() {
+	watchCacheEvent := &watchCacheEvent{
+		Type:   watch.Bookmark,
+		Object: w.newFunc(),
+	}
+
+	w.Lock()
+	defer w.Unlock()
+	if err := w.versioner.UpdateObject(watchCacheEvent.Object, w.resourceVersion); err != nil {
+		return
```

Comment:
Let's log error to an appropriate log level.

Reply:
done